chain-impl-mockchain/src/ledger/recovery.rs

```diff
@@ -1120,14 +1120,19 @@ fn unpack_entries(codec: &mut Codec<&[u8]>) -> Result<Vec<EntryOwned>, ReadError
     Ok(res)
 }
 
+impl SerializedSize for Ledger {
+    fn serialized_size(&self) -> usize {
+        unimplemented!()
```

Comment:
what's the problem with this?

Reply:
I have not added implementation for the `Ledger` type at this moment because it needs a huge effort to implement it for most of dependent types. Thus it is not actually used for the `Ledger` type I left it `unimplemented!()`.